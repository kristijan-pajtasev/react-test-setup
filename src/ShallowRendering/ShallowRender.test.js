import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow'; // ES6
import ComponentA from "./ComponentA";
const renderer = new ShallowRenderer();

it('renders correctly', () => {
    const tree = renderer
        .render(<ComponentA />)
    expect(tree).toMatchSnapshot();
});

it('renders correctly other', () => {
    const tree = renderer
        .render(<ComponentA />)
    expect(tree).toMatchSnapshot();
});