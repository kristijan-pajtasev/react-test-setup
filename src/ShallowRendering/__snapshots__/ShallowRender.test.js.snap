// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <ComponentB />
</div>
`;

exports[`renders correctly other 1`] = `
<div>
  <ComponentB />
</div>
`;
